var/spack/repos/builtin/packages/py-iterative-stats/package.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyIterativeStats(PythonPackage):
+    """Bacis iterative statistics implementation."""
+
+    pypi = "iterative-stats/iterative-stats-4.0.0.zip"
```

Comment:
```suggestion
    pypi = "iterative-stats/iterative_stats-0.0.4.tar.gz"
```
There is no zip file on PyPI